src/components/views/emojipicker/EmojiPicker.tsx

```diff
@@ -266,6 +271,15 @@ class EmojiPicker extends React.Component<IProps, IState> {
                         return categoryElement;
                     }) }
                 </AutoHideScrollbar>
+                {
+                    (this.state.filter) &&
+                        <AccessibleButton
+                            kind="link"
+                            onClick={() => this.reactWith(this.state.filter)}
+                        >
+                            React with "{ this.state.filter }"
```

Comment:
This should get translated using the `_t()` function. You can take a look at https://github.com/matrix-org/matrix-react-sdk/blob/develop/src/TextForEvent.tsx, it contains a bunch of code doing this, which you can use as an example

Reply:
After doing that change you will also need to run `yarn i18n` to update the translation files

Reply:
Fixed in 86632f8749. Thanks!

Reply:
np!